src/pieces/api/config.py

```diff
@@ -1,16 +1,17 @@
 import pieces_os_client as pos_client
-import importlib.resources
 from pathlib import Path
 import platform
 import os
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__))) # same as used in django!
 
+# Define the directory path
+pieces_data_dir = Path.home() / 'Documents' / 'com.pieces.os'/ 'cli-agent'
```

Comment:
This will be slightly different for macos/windows/linux, for now I would just create a generic folder called com.pieces.cli_agent in a directory that applications tend to store long term data